core/src/lib.rs

```diff
@@ -75,6 +74,11 @@ pub mod __reexports {
 	pub use async_trait::async_trait;
 	pub use serde;
 	pub use serde_json;
+	pub use tracing;
```

Comment:
please revert these, we don't want to re-export tracing and tokio from the core crate.